HOLMS.Messaging/RabbitConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace HOLMS.Messaging {
+    public class RabbitConfiguration {
```

Comment:
I have some reservations about calling this the `RabbitConfiguration`. The whole point of this library is to be sugar on top of RabbitMQ, so that (1) it's easier to use, and more constrained (e.g. in how we subscribe to things, how we serialize and deserialize messages), and (2) we enforce a uniform style of testing, including providing a standard set of mocks, on top of the raw rabbit driver (which seems to treat C# as a bit of an afterthought).

AMQP, which is the protocol rabbitmq uses, was intended to be pretty portable and not tied to a particular broker/server (e.g. rabbitmq). I'm not sure how well that goal was achieved in practice since we have to use the specific driver for the broker (RabbitMQ), not "the generic C# AMQP driver". I'm ambivalent on whether this class should be called "RabbitConfiguration" or just "MessagingConfiguration" or "MessagingConnectionConfiguration". 

I'll leave this one to you. I like the idea of a more generic/abstract messaging layer, but in the end, if Rabbit is all we're supporting, just leave it and move on. There is such a thing as too many layers.